tools/travis-test.sh

```diff
@@ -75,6 +75,7 @@ run_small_tests() {
   SMALL_SUMMARIES_DIRS=${BASE}/_build/test/logs/ct_run*
   SMALL_SUMMARIES_DIR=$(choose_newest_directory ${SMALL_SUMMARIES_DIRS})
   ${TOOLS}/summarise-ct-results ${SMALL_SUMMARIES_DIR}
+  DEBUG=1 ./rebar3 codecov analyze
```

Comment:
I think, it'd be better to call this in `.travis.yml` just before sending the report to codecov.

Reply:
or at least under the `$COVER_ENABLED` condition as in the removed `run_small_tests_cover_report` function.

Reply:
I'd go with the first option